apps/api/src/domain/models/audio-item/entities/transcript-item.entity.ts

```diff
@@ -0,0 +1,98 @@
+import { NestedDataType, NonEmptyString, NonNegativeFiniteNumber } from '@coscrad/data-types';
+import { isNumberWithinRange } from '@coscrad/validation-constraints';
+import { InternalError } from '../../../../lib/errors/InternalError';
+import { Maybe } from '../../../../lib/types/maybe';
+import { NotFound } from '../../../../lib/types/not-found';
+import { DTO } from '../../../../types/DTO';
+import { ResultOrError } from '../../../../types/ResultOrError';
+import { MultiLingualText } from '../../../common/entities/multi-lingual-text';
+import { Valid } from '../../../domainModelValidators/Valid';
+import { isNullOrUndefined } from '../../../utilities/validation/is-null-or-undefined';
+import BaseDomainModel from '../../BaseDomainModel';
+import { InvalidTimestampOrderError } from '../errors';
+
+// We can change this later
+type MediaTimestamp = number;
+
+export class TranscriptItem extends BaseDomainModel {
+    @NonNegativeFiniteNumber({
+        label: 'in point',
+        description: 'starting time stamp',
+    })
+    readonly inPoint: MediaTimestamp;
+
+    @NonNegativeFiniteNumber({
+        label: 'out point',
+        description: 'ending time stamp',
+    })
+    readonly outPoint: MediaTimestamp;
+
+    @NestedDataType(MultiLingualText, {
```

Comment:
We may want `MultiLingualText` to be a `CoscradDataType`. This could then be mapped to its own custom form element instead of working with nested form generation. We decide how to make this tradeoff when working on UX support.

Reply:
Let's do that on [this story](https://www.pivotaltracker.com/story/show/184376474).